modules/core/shared/src/main/scala/lucuma/core/math/Constants.scala

```diff
@@ -60,8 +55,7 @@ object Constants {
    */
 
   /** Speed of light in meters per second. Exact. */
-  val SpeedOfLight: Quantity[Int, MetersPerSecond] =
-    speedOfLightInVacuum[Int]
+  val SpeedOfLight: Quantity[Int, MetersPerSecond] = ???
```

Comment:
@erikerlandson dumb question of the day :) I couldn't figure out what to put on the rhs here, or at least the necessary `import`s to make it happen. In fact, I couldn't even summon `constant[coulomb.rational.Rational, SpeedOfLight]` without it complaining about a lack of conversion from `Rational` to itself.

Reply:
the `constant` method assumes a non-truncating value conversion, so probably you will need to do something like this:
```scala
import coulomb.units.constants.{*, given}
val SpeedOfLight = constant[Double, SpeedOfLight].tToValue[Int]
```

because I made truncating conversions explicit-only via `tToValue`

Reply:
I sort of assumed anyone who was in the business of using the kind of values defined in `coulomb.units.constants` would be wanting fractional value types

Reply:
D'oh, that's was a dumb one. Thanks, works nicely with spire `Rational` instead of `Double.`

Yeah, not sure how this particular API is used. I'll keep an eye out for it.